lib/core-common/src/presets.ts

```diff
@@ -79,15 +79,15 @@ export const resolveAddonName = (configDir: string, name: string) => {
 
   try {
     return {
-      name: resolveFrom(configDir, join(name, 'preset')),
+      name: resolveFrom(configDir, `${name}/preset`),
       type: 'presets',
     };
     // eslint-disable-next-line no-empty
   } catch (err) {}
 
   try {
     return {
-      name: resolveFrom(configDir, join(name, 'register')),
+      name: resolveFrom(configDir, `${name}/register`),
```

Comment:
🚨 This is a change to a non-test file.  The call to `resolveFrom` is expecting to resolve a node module, which always use forward slashes (it helped me to think about calls to `require()`).  Without this change, the mock of `resolve-from` in the tests would get windows paths, which it didn't know how to handle.  Also, I think that this is more-correct from a non-testing standpoint as well.